common/common.go

```diff
@@ -305,10 +317,16 @@ func LogTitle(logType LogType, v ...interface{}) {
 
 // SetLogType internal, used in flow package
 func SetLogType(logType LogType) {
-	log.SetFlags(0)
+//	log.SetFlags(0)
```

Comment:
Why?

Reply:
I will recheck this was done for the reuse of the existing logger and separate logs from stats and ngic.

Reply:
I don't understand why do you want to separate logs. Also you can't change library files to satisfy one example.

Reply:
We don't wants the stats in the logs as it's too clumsy so. had separate logger earlier. this was done as per the previous PR comment  "reuse existing logger and modify as needed ".

Reply:
also, other examples are continue to use the logging as was earlier